ffi/framebuffer_android.lua

```diff
@@ -70,29 +90,48 @@ function framebuffer:_updateWindow()
     android.lib.ANativeWindow_unlockAndPost(android.app.window);
 end
 
-function framebuffer:refreshFullImp()
+function framebuffer:refreshFullImp(x, y, w, h)
     self:_updateWindow()
-    update_eink(full)
+    if has_eink_full_support then
+        updatePartial(ntx_full, 50, x, y, w, h)
```

Comment:
Out of curiosity, what's the delay necessary for?

Reply:
no idea, copied from @char11 patch.

Reply:
I took it from the original code. I checked, with delay 0 also works. Sorry but I'm not a developer.

Reply:
In that case I'd say let's get rid of it! :-D